benchexec/test_cgroups_pytest.py

```diff
@@ -0,0 +1,60 @@
+# This file is part of BenchExec, a framework for reliable benchmarking:
+# https://github.com/sosy-lab/benchexec
+#
+# SPDX-FileCopyrightText: 2007-2020 Dirk Beyer <https://www.sosy-lab.org>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import logging
+import subprocess
+import pytest
+
+from benchexec import check_cgroups
+
+logging.disable(logging.CRITICAL)
+
+class TestCheckCgroups:
```

Comment:
Is it idiomatic for pytest to use classes for tests?

Reply:
There are no strict rules regarding organizing tests into modules vs classes. It's all a personal choice in my opinion

Reply:
What do you think for this case?

Reply:
As this test is comparatively small in size, it would be better to use classes according to me